### DISCUSSION TITLE
輸入法多個候選字，自定義選擇鍵

### DISCUSSION THREAD

**Discussion by sdnian** [fcitx/fcitx5#426] [Q&A] [answered]
在 4 版，輸入法如果有多個候選字，可以用 Choose 定義按鍵，而非數字鍵選擇。請問在 5 版，這個功能定義的關鍵字是什麼? 因為我用的大易輸入法，數字鍵是輸入的字，不能用於選字。

另外，如果自己新增輸入法，輸入法的 icon 檔案要放在哪裡?

**Comment by wengxt** [comment, marked as answer]
https://github.com/fcitx/fcitx5-table-extra/blob/3fe4137abb8fc94a8d62431422b837cac25dcfed/tables/easy-large.conf.in#L22

一个例子

图标放在系统图标主题一样的地方，例如用户级 ~/.local/share/icons/hicolor/，也可以参考fcitx自己的图标装在了哪

**Reply by sdnian** [reply]
問題解決了，感謝!
